package.json

```diff
@@ -45,7 +45,9 @@
     "test": "karma start ./karma.conf.js --single-run",
     "test:travis": "karma start ./karma.conf.js --single-run",
     "test:dev": "karma start ./karma.conf.js --no-single-run --auto-watch",
-    "test:cov": "karma start ./karma.conf.js --single-run --reporters coverage"
+    "test:cov": "karma start ./karma.conf.js --single-run --reporters coverage",
+    "pregh-pages": "git fetch origin",
+    "gh-pages": "git checkout gh-pages && git rebase origin/master --force-rebase && npm run build-demos && git add . && git commit --amend --no-edit && git push origin gh-pages --force && git checkout master"
```

Comment:
It is much easier if you look at it as:
```js
git checkout gh-pages
git rebase origin/master --force-rebase
npm run build-demos
git add .
# Overwrite previous commit (which is `Build`) with fresh new stuff
git commit --amend --no-edit
# Force-pushing
git push origin gh-pages --force
# Go back to master
git checkout master
```

This is a list you usually run to manually publish `gh-pages`. Maybe it is good to add `git reset --hard origin/gh-pages` too. And stash/unstash for safety.